spring-integration-scripting/src/main/java/org/springframework/integration/scripting/jsr223/AbstractScriptExecutor.java

```diff
@@ -61,11 +61,6 @@ protected AbstractScriptExecutor(String language) {
 	}
 
 	@Override
-	public Object executeScript(ScriptSource scriptSource) {
-		return this.executeScript(scriptSource, null);
-	}
-
-	@Override
 	public Object executeScript(ScriptSource scriptSource, Map<String, Object> variables) {
```

Comment:
??? See `ScriptExecutor`:
```
default Object executeScript(ScriptSource scriptSource) {
	return executeScript(scriptSource, null);
}
```
I've just moved the impl to the interface.
Semantically nothing has been changed for user.